### PR TITLE
Sorting files by components and services

### PR DESCRIPTION
**Pull request by bpeter340**
This should make the project look more organized and appealing.

### CODE REVIEW

**Comment by bpeter340** [comment]
This branch was created to organize the project files. Test files were also deleted since they are not necessary for a functioning program, however, they can always be regenerated by angular CLI later.

MERGE!

**Comment by KDRaypole** [comment]
I believe we should keep the tests. There is no point in removing test coverage when it isn't hurting anything. These specs will be important later when we decide to make this go live, we can hook up a CI to run our automation tests and ensure we don't disrupt anything.

> Test files were also deleted since they are not necessary for a functioning program

Necessary, no. But extremely Important for keeping our program working.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { LoginComponent } from './login/login.component'
-import { CallbackComponent } from './callback/callback.component'
-import { PlaylistComponent } from './playlist/playlist.component'
-import { SettingsComponent } from './settings/settings.component'
+import { LoginComponent } from './components/login/login.component'
```
Comment (KDRaypole) [inline src/app/app-routing.module.ts:3]: I noticed, when starting this project, that angular doesn't handle file organization the same as Ionic did. I don't know if this was by choice of the angular developers but I'm on board with filing components. 👍

Comment (bpeter340) [reply src/app/app-routing.module.ts:3]: So if I'm understanding you correctly, you're on board for sorting components, but not for temporarily deleting the test files. If so I can revert back to my first commit then merge to master.

Comment (KDRaypole) [reply src/app/app-routing.module.ts:3]: yessir

**Pull request merged**